lib/exercism/user_exercise.rb

```diff
@@ -11,7 +11,7 @@ class UserExercise < ActiveRecord::Base
       .where('views.user_id': user.id)
       .where('views.last_viewed_at > ?', 30.days.ago).order('views.last_viewed_at DESC')
   }
-  scope :current, -> { where(archived: false).where.not(iteration_count: 0).order('language, id ASC') }
+  scope :current, -> { where(archived: false).where.not(iteration_count: 0).order('last_iteration_at DESC') }
```

Comment:
I think it makes sense to sort them like this on the dashboard, but less so on the profile page, where it will now mix things together.

That said, I'm not really opposed to trying it and seeing whether anyone complains.

If y'all think that's fine, let's just merge this.